source/core/src/main/com/csse3200/game/inventory/items/AbstractItem.java

```diff
@@ -122,4 +128,9 @@ public void add(int n) {
      * @return whether there are none of this item remaining
      */
     public boolean isEmpty() {return quantity == 0;}
-}
\ No newline at end of file
+
+    public void getTexture(String texture) {
+        this.itemTexture = new Texture(texture);
```

Comment:
Create a setter which takes in a texture path and creates a new texture - getter should then take no arguments and should just return that texture (and maybe throw an error if the texture has not been initialised - ie it's still null).